lib/write.ml

```diff
@@ -175,7 +175,7 @@ let write_normal_float_prec significant_figures ob x =
   if float_needs_period s then
     Buffer.add_string ob ".0"
 
-let write_float_prec significant_figures ob x =
+let write_float_prec significant_figures ob x = (* used by atdgen *)
```

Comment:
Please add these comments on the line above, inline comments are nearly always a pain to deal with.